x-pack/plugin/sql/src/main/resources/org/elasticsearch/xpack/sql/plugin/sql_whitelist.txt

```diff
@@ -130,4 +130,9 @@ class org.elasticsearch.xpack.sql.expression.function.scalar.whitelist.InternalS
   String  space(Number)
   String  substring(String, Number, Number)
   String  ucase(String)
+  
+#
+# Casting
+#
+  def convert(Object, String)
```

Comment:
I have a slight preference to name it `cast` which is the main name for SQL, but convert is also good.

Reply:
+1 on cast for me as well.